.github/workflows/eaa_kbc.yml

```diff
@@ -25,34 +25,47 @@ jobs:
     steps:
       - name: Install rats-tls dependencies
         run: |
-          apt-get update && apt-get install -y autoconf gcc g++ make wget git cmake \
+          sudo apt-get update;
+          sudo apt-get install -y autoconf gcc g++ make wget git cmake \
           libseccomp-dev binutils-dev libprotoc-dev protobuf-compiler \
           pkg-config libssl-dev openssl libtool file curl gnupg;
 
-          wget https://github.com/protobuf-c/protobuf-c/archive/v1.3.1.tar.gz &&\
-          tar -zxvf v1.3.1.tar.gz && cd protobuf-c-1.3.1 && ./autogen.sh && \
-          ./configure && make && make install
+          sudo wget https://github.com/protobuf-c/protobuf-c/archive/v1.3.1.tar.gz;
+          sudo tar -zxvf v1.3.1.tar.gz;
+          cd protobuf-c-1.3.1 && sudo ./autogen.sh;
+          sudo ./configure;
+          sudo make;
+          sudo make install;
 
-          wget https://download.01.org/intel-sgx/sgx-linux/2.14/as.ld.objdump.gold.r3.tar.gz && \
-          tar -zxvf as.ld.objdump.gold.r3.tar.gz && cp -rf external/toolset/ubuntu18.04/* /usr/local/bin/ && \
-          rm -rf external && rm -rf as.ld.objdump.gold.r3.tar.gz
-
-          wget https://download.01.org/intel-sgx/sgx-linux/2.14/distro/ubuntu18.04-server/sgx_linux_x64_sdk_2.14.100.2.bin;
-          chmod +x sgx_linux_x64_sdk_2.14.100.2.bin &&  echo -e 'no\n/opt/intel\n' | ./sgx_linux_x64_sdk_2.14.100.2.bin;
-          rm -rf sgx_linux_x64_sdk_2.14.100.2.bin;
+          sudo wget https://download.01.org/intel-sgx/sgx-linux/2.14/as.ld.objdump.gold.r3.tar.gz;
+          sudo tar -zxvf as.ld.objdump.gold.r3.tar.gz;
+          sudo cp -rf external/toolset/ubuntu18.04/* /usr/local/bin/ && \
+          sudo rm -rf external && sudo rm -rf as.ld.objdump.gold.r3.tar.gz
 
+          sudo wget https://download.01.org/intel-sgx/sgx-linux/2.14/distro/ubuntu18.04-server/sgx_linux_x64_sdk_2.14.100.2.bin;
+          sudo chmod +x sgx_linux_x64_sdk_2.14.100.2.bin;
+          echo -e 'no\n/opt/intel\n' | sudo ./sgx_linux_x64_sdk_2.14.100.2.bin;
```

Comment:
Might be worth adding a comment to explain what "no" means. Presumably `/opt/intel` is the prefix?